test/services/supported_ssg_updater_test.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+# A class to test the SupportedSsgUpdater service
+class SupportedSsgUpdaterTest < ActiveSupport::TestCase
+  context 'fallback file' do
+    setup do
+      @supported_file = ::Tempfile.new('suported_ssg_file')
+      @supported_file.write(File.read('test/fixtures/files/supported_ssg_test.yaml'))
+      @supported_file.rewind
+
+      @fallback_file = ::Tempfile.new('fallback_ssg_file')
+    end
```

Comment:
When you create files, you should also clean them up in a `teardown` block. See other tests for examples.